README.md

```diff
@@ -1,7 +1,7 @@
 #Pome
 Pome stands for **Po**stgres **Me**trics.
 Pome is a PostgreSQL Metrics Dashboard to keep track of the health of your database.
-This project is at a very early stage and there are a lot of missing features,
+This project is at a very early stage and there are a lot of missing features
```

Comment:
Is the comma incorrect here? "I'm hoping to be able to make the project progress quickly" is a separate clause so I think it's ok to use a comma.